lib/hammer_cli_katello/sync_plan.rb

```diff
@@ -32,9 +33,11 @@ class InfoCommand < HammerCLIKatello::InfoCommand
     class CreateCommand < HammerCLIKatello::CreateCommand
       option "--interval", "INTERVAL", _("how often synchronization should run"),
              :format => HammerCLI::Options::Normalizers::Enum.new(
-               %w(hourly daily weekly)
+               ["hourly", "daily", "weekly", "custom cron"]
```

Comment:
Since we need added validation for cron_expression, I think we can also remove this Enum format completely and validate the interval in Katello. Right now, there is validation, but it's not clear what the options are from the error message (note, I removed this format to achieve the following output):

```
$ hammer sync-plan create --name foo3 --organization-id 1 --interval weeklys --sync-date 2018-09-11 --enabled 1
Could not create the sync plan:
  Interval not set correctly
```

Reply:
I will go ahead and remove the enum format.. Are you suggesting a change in validation message somewhere to prompt the hammer user of valid options?

Reply:
Not only hammer users, but API users in general. Almost all of our error messages are generated by the API; this is a special case for some reason, and I don't think it needs to be.

Reply:
Can we change the name of 'custom cron' to 'custom-cron'. 
Reasons : 
1. Autocomplete is hammer shell show 'custom cron', if you see hammer shell whenever there are two names are joined using '-'  e.g. 'sync-plan', 'organization-id'
below example explains more too 


```
hammer> sync-plan create --name 'test1' --interval custom cron --organization-id 1 --enabled 1 --sync-date '2016-10-01 01:00:00'
Could not create the sync plan:
  Error: Option '--interval': Value must be one of 'hourly', 'daily', 'weekly', 'custom cron'..
  
  See: ' sync-plan create --help'.
hammer> sync-plan create --name 'test1' --interval daily --organization-id 1 --enabled 1 --sync-date '2016-10-01 01:00:00'Sync plan created.
```
2. As per above example the for daily get autocomplete no need to user to convert it to 'daily', same we can do custom-cron instread of 'custom cron'

@akofink , @sjha4  Let me know what you think ?

Reply:
I don't think we should change it. Hyphens are used for subcommands and options (i.e. --organization-id), but not for values (i.e. 'Default Organization'). Values with spaces may use quotes or escape the space (i.e. `'Default Organization'` or `Default\ Organization`). This is common throughout the different hammer resources.

Reply:
Ok,  make sense